autoload/neomake.vim

```diff
@@ -42,6 +42,13 @@ function! neomake#GetJob(job_id) abort
     return s:jobs[a:job_id]
 endfunction
 
+function! neomake#GetErrors(bufnr) abort
+    if !a:bufnr
+        return s:current_errors['project']
+    endif
+    return get(s:current_errors['file'], a:bufnr, {})
+endfunction
+
```

Comment:
The (public) API here should be better.. 
See https://github.com/vim-airline/vim-airline/pull/1275#issuecomment-271122146 for some discussion / link.

As for this PR: it is only used once, so keep it inlined for now?!